.github/workflows/slack-eod-bot.yaml

```
@@ -0,0 +1,15 @@
+name: Slack EOD Reminder Bot
+
+on:
+  schedule:
+    - cron: "30 13 * * *" # 7:00pm IST
+  workflow_dispatch:
+
+jobs:
+  run-slack-eod-reminder:
+    uses: coronasafe/leaderboard/.github/workflows/slack_eod_reminder.yaml@slack-eod-bot
```

Target branch will be updated once tested

```suggestion
    uses: coronasafe/leaderboard/.github/workflows/slack_eod_reminder.yaml@main
```